### DISCUSSION TITLE
How to build TreeSelect with MultiSelect options?

### DISCUSSION THREAD

**Discussion by Varun-19** [mantinedev/mantine#6538] [Q&A]
how to build a nested menu with multi select and that can include checkbox as well for the options

<img width="278" alt="Screenshot 2024-07-17 at 1 09 54 PM" src="https://github.com/user-attachments/assets/765f1f05-5520-4f54-8660-2bc37b7174a9">

**Comment by rtivital** [comment]
Currently, Mantine does not provide such feature, you can use a third-party library for this.
